src/main/config/localhost/data.sql

```diff
@@ -6,5 +6,4 @@ INSERT INTO tutor (fname,lname,email) VALUES ('Scott','Chow','scottpchow@example
 INSERT INTO tutor (fname,lname,email) VALUES ('Zach','Sisco','zachsisco@example.org');
 INSERT INTO tutor (fname,lname,email) VALUES ('Yinon','Rousso','yinonRousso@example.org');
 INSERT INTO tutor (fname,lname,email) VALUES ('Kate','Perkins','kateperkins@example.org');
-INSERT INTO tutor (fname,lname,email) VALUES ('George','Kripac','Georgekripac@example.org');
-INSERT INTO tutor_assignment (course_offering_id,tutor_id) VALUES (1,1);
\ No newline at end of file
+INSERT INTO tutor (fname,lname,email) VALUES ('George','Kripac','Georgekripac@example.org');
```

Comment:
Why did we delete the tutor_assignment?

Reply:
I had removed all of the SQL inserts to test the upload button. I forgot to add this back.